owncloudTestUtil/src/main/java/com/owncloud/android/testutil/OCServerInfo.kt

```diff
@@ -3,6 +3,8 @@ package com.owncloud.android.testutil
 import com.owncloud.android.domain.server.model.AuthenticationMethod
 import com.owncloud.android.domain.server.model.ServerInfo
 
+const val OC_BASE_URL = "https://demo.owncloud.com"
+
 val OC_SERVER_INFO = ServerInfo(
     authenticationMethod = AuthenticationMethod.BASIC_HTTP_AUTH,
     baseUrl = "https://demo.owncloud.com",
```

Comment:
Why do you need `OC_BASE_URL` and not `OC_SERVER_INFO.baseUrl`?

Reply:
There are some tests that have nothing to do with server info objects, e.g. https://github.com/owncloud/android/blob/new_arch/complete_login_flow/owncloudData/src/androidTest/java/com/owncloud/android/data/authentication/datasources/implementation/OCLocalAuthenticationDataSourceTest.kt#L263

Reply:
Ok, i only tell you that in order not to create several constants with the same value.

Reply:
I can change the constant value so that has a different value but the point here is that are used for different tests, so it doesn't matter that much.